API Test App/ConcertAPI/concert/views.py

```diff
@@ -103,4 +105,11 @@ def user_detail(request, pk):
         user.delete()
         return HttpResponse(status=204)
 
+class Test(unittest.TestCase):    
+    def test_get_a_concert(request,pk):
+        expected_result = ''
+        unittest.TestCase.assertEqual(concert_detail(request,pk), '')
```

Comment:
Yes! I was going to integrate testing into the application itself but found a better way. I forgot to delete this one 😃